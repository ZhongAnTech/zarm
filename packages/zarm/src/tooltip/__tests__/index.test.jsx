import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Tooltip from '../index';

describe('Tooltip', () => {
  it('renders correctly', () => {
    jest.useFakeTimers();
    const wrapper = mount(
      <Tooltip direction="top" title="hello">
        <span>foo</span>
      </Tooltip>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
    jest.runAllTimers();
    wrapper.unmount();
  });
});
