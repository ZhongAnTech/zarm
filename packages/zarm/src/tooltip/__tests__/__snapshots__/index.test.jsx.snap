// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders correctly 1`] = `
<Tooltip
  direction="top"
  hasArrow={true}
  onVisibleChange={[Function]}
  prefixCls="za-tooltip"
  title="hello"
>
  <span>
    foo
  </span>
</Tooltip>
`;
